cmd/security-profiles-operator-init/main.go

```diff
@@ -0,0 +1,233 @@
+	operatorLinkEnvKey       = "OPERATOR_SYMLINK"
+	operatorUID              = 2000
+	operatorGID              = 2000
+)
+
+var setupLog = ctrl.Log.WithName("setup")
+
+func main() {
+	ctrl.SetLogger(klogr.New())
+
+	app := cli.NewApp()
+	app.Name = config.OperatorName + "-init"
+	app.Usage = "Kubernetes Seccomp Operator Setup"
+	app.Description = "Setup script for the Kubernetes Seccomp Operator."
+	app.Version = version.Get().Version
+	app.Action = run
+	app.Commands = cli.Commands{
+		&cli.Command{
+			Name:    "version",
+			Aliases: []string{"v"},
+			Usage:   "display detailed version information",
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:    jsonFlag,
+					Aliases: []string{"j"},
+					Usage:   "print JSON instead of text",
+				},
+			},
+			Action: func(c *cli.Context) error {
+				v := version.Get()
+				res := v.String()
+				if c.Bool(jsonFlag) {
+					j, err := v.JSONString()
+					if err != nil {
+						return errors.Wrap(err, "unable to generate JSON from version info")
+					}
+					res = j
+				}
+				print(res)
+				return nil
+			},
+		},
+	}
+	app.Flags = []cli.Flag{
+		&cli.StringFlag{
+			Name:  "path",
+			Usage: "mount path for default profiles to bootstrap",
+		},
+	}
+
+	if err := app.Run(os.Args); err != nil {
+		setupLog.Error(err, "running security-profiles-operator")
+		os.Exit(1)
+	}
+}
+
+func run(c *cli.Context) error {
+	v := version.Get()
+	setupLog.Info(
+		"starting security-profiles-operator-init",
+		"version", v.Version,
+		"gitCommit", v.GitCommit,
+		"gitTreeState", v.GitTreeState,
+		"buildDate", v.BuildDate,
+		"goVersion", v.GoVersion,
+		"compiler", v.Compiler,
+		"platform", v.Platform,
+	)
+
+	if err := v1alpha1.AddToScheme(scheme.Scheme); err != nil {
+		return errors.Wrap(err, "setting up client")
+	}
+
+	clientset, err := getClient()
+	if err != nil {
+		return errors.Wrap(err, "getting REST client")
+	}
+
+	profiles, err := getProfiles(c.String("path"))
+	if err != nil {
+		return errors.Wrap(err, "getting profiles")
+	}
+
+	// Create SeccompProfile resources
+	ctx := context.Background()
+	for name, sp := range profiles {
+		result := v1alpha1.SeccompProfile{}
+		err = clientset.Post().
+			Namespace(operatorNamespace).
+			Resource(resourceName).
+			Body(sp).
+			Do(ctx).
+			Into(&result)
+		if err != nil && !apierrors.IsAlreadyExists(err) {
+			return errors.Wrap(err, "creating SeccompProfile")
+		}
+		if apierrors.IsAlreadyExists(err) {
+			setupLog.Info(fmt.Sprintf("skipping creation of existing seccomp profile %s", name))
+		} else {
+			setupLog.Info(fmt.Sprintf("successfully created seccomp profile %s", name))
+		}
+	}
+
+	// Set up directories for non-root access
+	kubeletSeccompRoot := os.Getenv(kubeletSeccompRootEnvKey)
+	operatorRoot := os.Getenv(operatorRootEnvKey)
+	operatorLink := os.Getenv(operatorLinkEnvKey)
+	f, err := os.Lstat(kubeletSeccompRoot)
+	if os.IsNotExist(err) || !f.IsDir() {
+		if err = os.Mkdir(kubeletSeccompRoot, 0744); err != nil {
```

Comment:
More a style than a functional nit:
```suggestion
		if err = os.Mkdir(kubeletSeccompRoot, os.FileMode(0o744)); err != nil {
```

Reply:
done